.github/workflows/wheel.yml

```diff
@@ -6,6 +6,9 @@ on: [push, pull_request]
 jobs:
   wheels:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7]
```

Comment:
Please add 3.8 and 3.9?  This is the script the will populate wheels for each python version, yes?

Reply:
Done.